### DISCUSSION TITLE
Is there a way to get all NFL games without adding each individual team?

### DISCUSSION THREAD

**Discussion by goaliedude3919** [vasqued2/ha-teamtracker#115] [Q&A]
From what I've seen, it doesn't look like this is currently possible but I wanted to make sure I wasn't missing something. The wildcard option for team sports says that it's unpredictable when multiple games are at the same time because it will only display the "current game" which would obviously be confusing if multiple games are at the same time.

**Comment by vasqued2** [comment]
You need to set up a sensor for each team.

If you look at the sensor.yaml file in [this repository](https://github.com/kbrown01/SportStandingsScores), you will find someone has already created them at the end of the file.
